src/MongoDB/Session.c

```diff
@@ -390,6 +390,24 @@ static PHP_METHOD(Session, endSession)
 	mongoc_client_session_destroy(intern->client_session);
 } /* }}} */
 
+/* {{{ proto void MongoDB\Driver\Session::isInTransaction(void)
+   Returns whether a multi-document transaction is in progress */
+static PHP_METHOD(Session, isInTransaction)
+{
+	php_phongo_session_t* intern;
+	SUPPRESS_UNUSED_WARNING(return_value_ptr)
+	SUPPRESS_UNUSED_WARNING(return_value_used)
+
+	intern = Z_SESSION_OBJ_P(getThis());
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	RETURN_BOOL(!!mongoc_client_session_in_transaction(intern->client_session));
```

Comment:
Is the quick bool cast with `!!` necessary? [The function](https://github.com/mongodb/mongo-c-driver/blob/ba4a1261b31be4aa103a601f63553191c1a65b04/src/libmongoc/src/mongoc/mongoc-client-session.c#L772) returns a `bool` and [`_mongoc_client_session_in_txn()`](https://github.com/mongodb/mongo-c-driver/blob/ba4a1261b31be4aa103a601f63553191c1a65b04/src/libmongoc/src/mongoc/mongoc-client-session.c#L909) uses `true` and `false` return values.

I would imagine there are a few other places where we trust libmongoc's `bool` return values.

Reply:
Removed the ``!!``.